includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -195,4 +203,50 @@ private function isWikiaOrgCommunity() {
 
 		return wfGetBaseDomainForHost( parse_url( $wgServer, PHP_URL_HOST ) ) === $wgWikiaOrgBaseDomain;
 	}
+
+
+	/**
+	 * External API request to IntentX
+	 */
+	public function getFandomShopDataFromIntentX() {
+		global $wgCityId, $wgMemc, $wgFandomShopMap, $wgFandomShopUrl;
+
+		// don't make api call if not fandom store community
+		if ( !array_key_exists( $wgCityId, $wgFandomShopMap ) ) {
+			return null;
+		}
+
+		// api for fandom store will be replaced with $wgFandomShopUrl
+		$uri = 'http://138.201.119.29:9420/ix/api/seo/v1/footer'; // https://shop.fandom.com/ix/api/seo/v1/footer 500 - Internal Server Error
+		$memcKey = wfMemcKey( self::MEMC_PREFIX_FANDOM_STORE, $wgCityId );
+
+		// do api request
+		$client = new Client( [
+			'base_uri' => $uri,
```

Comment:
Let's remember to replace `$url` with `$wgFandomShopUrl` :)

Reply:
done.